types/joi/joi-tests.ts

```diff
@@ -113,6 +113,12 @@ uriOpts = { scheme: expArr };
 
 // --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- ---
 
+var base64Opts: Joi.Base64Options = null;
```

Comment:
Why not combine these statements?

Reply:
Also this should be `let/const` not `var`.

Reply:
Agreed on `let/const` vs `var` but I kept consistent with all the other Options. Same with the separate statements instead of combining them.

Favored consistency in this case. Should I break from that and optimize?

Reply:
Eh, it just seems silly to me and `var` is effectively (not actually) deprecated so.

Reply:
I figure a separate pull request should be made to change all that over then, no?

I guess that is up to @Bartvds ?